tracetools/include/tracetools/tracetools.h

```diff
@@ -103,7 +103,7 @@
  * This is the preferred method over calling the underlying function directly.
  */
 #  define TRACETOOLS_TRACEPOINT_ENABLED(event_name) \
-  ros_trace_enabled_ ## event_name()
+  ros_trace_enabled_ ## event_name(void)
```

Comment:
This is not going to work (see build failures in GitHub CI), because these macros are used for both declaration and implementation. I'd need to split them up.

Reply:
If you wanted to, you could split out the other changes (or remove the changes involving macros above from this PR), since they should be fine:

- `ros_trace_compile_status()` in this file
- `tracetools/src/status_tool.c`
- `tracetools/src/tracetools.c`

Then I can look into dealing with the changes to macros separately.